ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -3215,21 +3215,24 @@ traffic_policies:
     counter_per_interface: < true | false >
   field_sets:
     ipv4:
-      < PREFIX FIELD SET NAME >:
-        - < IPv4 prefix 01>
-        - < IPv4 prefix 02>
-        - < IPv4 prefix 03>
+      - name: < PREFIX FIELD SET NAME >
+        prefix:
+          - < IPv4 prefix 01>
+          - < IPv4 prefix 02>
+          - < IPv4 prefix 03>
     ipv6:
-      < PREFIX FIELD SET NAME >:
-        - < IPv6 prefix 01>
-        - < IPv6 prefix 02>
-        - < IPv6 prefix 03>
+      - name: < PREFIX FIELD SET NAME >
+        prefix:
+          - < IPv6 prefix 01>
+          - < IPv6 prefix 02>
+          - < IPv6 prefix 03>
     ports:
-      < L4 PORT FIELD SET NAME >: "< vlan range >"
+      - name: < L4 PORT FIELD SET NAME >
+        vlan_range: "< vlan range >"
```

Comment:
I don't think this is vlan range. It must be a typo in the old documentation. This is port_range.

Reply:
updated.